pkgs/by-name/ng/ngrok/update.sh

```diff
@@ -31,4 +31,4 @@ get_download_info() {
     get_download_info darwin amd64
     get_download_info darwin arm64
 ) | jq --slurp 'map ({ (.sys): . }) | add' \
-    > pkgs/tools/networking/ngrok/versions.json
+    > pkgs/by-name/ng/ngrok/versions.json
```

Comment:
Instead of hard coding, you can use the relative reference of this file (`BASH_SOURCE[0]`).

Reply:
Good point. Quite a few update.sh files already use this, so I opted for `cd "$(dirname "${BASH_SOURCE[0]}")"`.